packages/react/src/themes/teams-dark/componentVariables.ts

```diff
@@ -15,5 +15,4 @@ export { default as Icon } from './components/Icon/iconVariables'
 export { default as Reaction } from './components/Reaction/reactionVariables'
 export { default as Alert } from './components/Alert/alertVariables'
 export { default as ProviderBox } from './components/Provider/providerBoxVariables'
```

Comment:
Great that we manage to remove all dark theme overrides, just double check the screener differences.

Reply:
There are lots of changes in dark theme that changes the background of the Dropdown. Make sure that we are not breaking something

Reply:
Looks good to me